.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Hello, I can see that you did ignore the `test.md` file. However, I think it has been cached because its still among the files in your repository.
I will suggest you also ignore the `node_modules` directory.

To fix the cached files issues, navigate into your project from the terminal and run this command `git rm -r --cached .`
After running that command, please stage your files, commit and make a push to GitHub again. Verify that the `test.md` file no longer exists in your repo before asking for a re-review.

Please note that this command `git rm -r --cached .` will clear all cached files and will not push any cached files to your remote repository.